### PR TITLE
Update Hyundai

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
hyundai

**Issue:** contributes toward #9072 

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from website header https://www.hyundainews.com
Color: Taken from logo `#002c5e`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -6135,9 +6135,9 @@
         },
         {
             "title": "Hyundai",
-            "hex": "002C5F",
-            "source": "https://en.wikipedia.org/wiki/File:Hyundai_Motor_Company_logo.svg",
-            "guidelines": "https://www.hyundai.pl/fileadmin/user_upload/media/logo/201607_HYU_Guideline_ENG_small.pdf"
+            "hex": "002C5E",
+            "source": "https://www.hyundainews.com",
+            "guidelines": "https://www.hyundainews.com/en-us/contact"
```
Comment (adamrusted) [inline _data/simple-icons.json:6140]:
```suggestion
            "source": "https://www.hyundai.com"
```

SVG icon available as part of the webfont used on their global homepage. Also don't think a contact page is an adequate guidelines link.

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (PratyushJayachandran) [reply _data/simple-icons.json:6140]: image
This is why I attached the contact page as that is how they want media inquiries to be made.

Comment (PratyushJayachandran) [reply _data/simple-icons.json]: Corrected the source and updated the icon

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/hyundai.svg [hunk 2]
```diff
@@ -1 +1 @@
...
... 3.252 4.622-.22 1.968-2.01 3.342-3.61 4.17-4.668 2.287-11.225 2.358-16.046.359C2.7 16.143.873 14.949.187 13.153c-.525-1.475.115-3.014 1.189-4.037C3.62 7.023 6.485 6.305 9.432 5.891c3.198-.399 6.565-.047 9.429.96.64.242 1.271.508 1.873.835v.001zM18.1 7.625c-1.912-.547-3.922-.875-6.055-.858-.99.015-1.706.96-2.11 1.764-.304.64-.654 1.405-.396 2.14a.974.974 0 0 0 .602.467c1.043.234 1.988-.188 2.924-.421 1.843-.617 3.649-1.32 5.225-2.584.068-.086.19-.172.129-.305-.068-.118-.205-.14-.32-.203zM8.16 7.25c-.053-.1-.19-.086-.297-.086-1.02.164-1.988.453-2.917.813-1.569.679-3.45 1.678-3.915 3.552-.289 1.436.716 2.646 1.828 3.396.22.1.457.414.701.18 1.25-2.694 2.566-5.403 4.57-7.738l.03-.117zm13.29 2.062c-.275-.188-.519-.461-.861-.492a.39.39 0 0 0-.168.116c-.37.843-.765 1.674-1.188 2.491-.921 1.82-1.972 3.615-3.359 5.215l-.022.11c.015.094.114.078.175.086.975-.093 1.874-.406 2.765-.71 1.637-.657 3.693-1.687 4.165-3.623.29-1.312-.563-2.397-1.508-3.194zm-6.938 4.137c-.122-.328-.465-.57-.808-.617-.83-.109-1.615.125-2.353.343-1.98.68-3.983 1.328-5.667 2.726-.075.07-.128.17-.09.28a.361.361 0 0 0 .212.157 21.01 21.01 0 0 0 5.865.921c.907.094 1.66-.53 2.117-1.256.411-.781.906-1.585.724-2.554z"/></svg>
```
Comment (adamrusted) [inline icons/hyundai.svg:1]: SVG available in the footer of the site gives [some differences](https://github.com/simple-icons/simple-icons/assets/1830867/27a77111-90c1-4030-9700-22623ddd4976). Would you mind taking another look?

Comment (PratyushJayachandran) [reply icons/hyundai.svg:1]: I took the logo from header(hyundai-logo-wide-blue.svg) and on comparing it is different from the footer(hyundai-logo-stacked-white.svg).

Which one should we take?

Comment (adamrusted) [reply icons/hyundai.svg:1]: The logo in the header seems to be part of an icon font, which isn't easy to decompile (without a browser plugin!). I would say the footer is a more _accessible_ source for people wanting to review in future.

File: icons/hyundai.svg
Comment (adamrusted) [inline icons/hyundai.svg:1]: I'm still seeing a _fair few_ [differences](https://github.com/simple-icons/simple-icons/assets/1830867/cdb28958-49d0-4ec3-ac0a-133d45b7eff5) between the footer image in red, and your PR in black @PratyushJayachandran. Could you have another go please? 🙏

**Review by adamrusted — CHANGES_REQUESTED** [review]

**Comment by adamrusted** [comment]
@PratyushJayachandran - able to fix this one, or do you need a hand?

**Comment by PratyushJayachandran** [comment]
> @PratyushJayachandran - able to fix this one, or do you need a hand?

Yeah. I'm not understanding the issue. It seems fine at my end. 😔

**Comment by adamrusted** [comment]
@PratyushJayachandran - I'm using [this file](https://www.hyundai.com/etc.clientlibs/hyundai-wwn3/clientlibs/main/resources/img/hyundai-logo-white.svg) as the source file. May just be a precision thing as the differences are _relatively_ minor.

**Comment by PratyushJayachandran** [comment]
> @PratyushJayachandran - I'm using [this file](https://www.hyundai.com/etc.clientlibs/hyundai-wwn3/clientlibs/main/resources/img/hyundai-logo-white.svg) as the source file. May just be a precision thing as the differences are _relatively_ minor.

This doesn't look like the source I have been using. I had used the header one initially and then switched to footer one.

I just used the icon you provided in the previous comment and redid it. Please check again @adamrusted

**Review by adamrusted — APPROVED** [review]

**Pull request merged**